### ISSUE TITLE
No feedback when running provider verification through pact-go daemon

### ISSUE THREAD

**Issue by bmarini** [pact-foundation/pact-go#55]
I ended up bypassing the daemon and shelling out to the ruby `pact-provider-verifier` directly because none of the verifier's output was being returned to me in my tests.

``` golang
func verifyProvider(t *testing.T, baseURL, setupURL string, pactURLS []string) {
	args := append(pactURLS,
		"--provider-base-url", baseURL,
		"--provider-states-setup-url", setupURL,
	)
	cmd := exec.Command("pact-provider-verifier", args...)

	out, err := cmd.CombinedOutput()
	if err != nil {
		t.Errorf("Provider verification failed: %s:\n\n%s\n", err, out)
	}
}
```

Running the verification as described in the README, I get something similar to this:

```
--- FAIL: TestPactProvider (0.10s)
pact_test.go:34: Error: provider verification failed:
exit status 1
```

**Comment by bmarini** [comment]
I get that this is a temporary issue until there is a native go implementation, and provider verification can run in the test process, with no need to call out to a separate binary, but running the provider verifier through the daemon currently is pretty useless since you don't see the test output.

**Comment by mefellows** [comment]
Thanks, would you be open to submitting A PR to make it better?

**Comment by bmarini** [comment]
I'd like to help out on the native go implementation of the v2 spec, is that in the works anywhere?

**Comment by bethesque** [comment]
We're working on a Rust implementation which will be used by all the other implementations using FFI. Until then, we're stuck with the separate process. Your help to improve the output would be greatly appreciated.

**Comment by bmarini** [comment]
Ok, I looked into this a bit more and I think its a matter of getting pact-go to ask for the json formatted version of the verifier output, and then packaging that up in a usable way back into the client code. I may take a stab at it.

**Comment by mefellows** [comment]
That JSON formatted output is a relatively new thing. Would love a PR (I'm lost for time atm).

If you're feeling really generous, I would appreciate any help on https://github.com/pact-foundation/pact-go/issues/31. This essentially acknowledges what you are talking about above, and proposes some solutions. 

The Reference is still some way away (although good progress is coming), but is definitely closer than re-implementing it in its entirety in Go, particularly when you consider v3 and v4 (draft) are on their way too.

**Comment by bmarini** [comment]
#57 should fix this :)

**Comment by mefellows** [comment]
Released v0.0.10
